openedx_external_enrollments/settings/test.py

```diff
@@ -6,6 +6,19 @@
 
 from .common import *  # pylint: disable=wildcard-import
 
+
+class DisableMigrations(dict):
```

Comment:
I think you don't need to add this. Ref: https://docs.djangoproject.com/en/2.2/ref/settings/#migration-modules @ivanvgh

Reply:
Nice catch!